addons/mail/static/src/chatter/web/mail_composer_form.js

```diff
@@ -27,6 +32,31 @@ export class MailComposerFormRenderer extends formView.Renderer {
             this.root,
             this.props.record.resId
         ]);
+
+        // Add the file dropzone on the mail composer.
+        this.notification = useService("notification");
+        this.uploadFiles = useFileUploader();
+        this.operations = useX2ManyCrud(() => {
+            return this.props.record.data["attachment_ids"];
+        }, true);
+        useDropzone(this.root, async event => {
+            const { files } = event.dataTransfer;
+            try {
+                const attachments = await this.uploadFiles("/web/binary/upload_attachment", {
+                    csrf_token: odoo.csrf_token,
+                    ufile: [...files],
+                    model: this.props.record.resModel,
+                    id: this.props.record.resId || 0,
+                });
+                await this.operations.saveRecord(attachments.map(attachment => {
+                    return attachment.id
+                }));
+            } catch {
+                this.notification.add(_t("Error while uploading your file. Please make sure to upload a file."), {
+                    type: "danger",
+                });
+            }
```

Comment:
Similarly to the small composer, shouldn't this be overridden in cloud storage to by-pass the limit and upload to the proper service?